src/pages/SharedPage.tsx

```diff
@@ -0,0 +1,158 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { getFolderRequest } from "../api/folderApi";
+import FolderInfo, {
+  folderInfoProps,
+} from "../components/FolderInfo/FolderInfo";
+import { linkCardProp } from "../components/LinkCard/LinkCardItem";
+import LinkCardList from "../components/LinkCard/LinkCardList";
+import SearchBar from "../components/SearchBar/SearchBar";
+
+const SharedPage = () => {
+  const [folderData, setFolderData] = useState<folderInfoProps>();
+  const [cardDataList, setCardDataList] = useState<linkCardProp[]>();
+
+  const getCardListProp = (dataList: any) => {
```

Comment:
```suggestion
  const parseCards = (cards: Card[]) => {
```
1. prop을 생성하기 위한 함수이지만 굳이 뒤에 prop을 붙일 필요는 없어 보여요. 
2. any보단 적절한 type을 선언하고 사용하는 습관을 들이는게 좋습니다. 그렇지 않으면 ts를 쓰는 의미가 점점 퇴색해요.